src/conv/invokers/impl_gemm_dynamic.cpp

```diff
@@ -68,6 +68,110 @@ float CallImplGemmDynamicForward(const miopen::Handle& handle,
     return elapsed;
 }
 
+float CallImplGemmDynamicForward(const miopen::Handle& handle,
```

Comment:
this function seems a overload of `CallImplGemmDynamicForward` used in fp32 with different function args.
I think we'd better use a single function, and add the `solver::TunableImplicitGemmGTCDynamic_t` args by default. since current fp32 version will not use magic_div, there would be a branch there. But yes ultimately we would update all dynamic asm igemm to utilize magic_div hence at that time a single function of `CallImplGemmDynamicForward` is reasonalbe.

Reply:
ok

Reply:
modified. Since legacy dynamic igemm kernel use the same API to call device function, modification is complicate.

Reply:
@carlushuang Could you pls re-reviews changes?